src/dotnet/commands/dotnet-add/dotnet-add-p2p/Program.cs

```diff
@@ -81,5 +118,10 @@ public static int Run(string[] args)
                 return 1;
             }
         }
+
+        private static string GetFrameworkDisplayString(NuGetFramework framework)
```

Comment:
I like it, particularly if we convert it to an extension method. There's a lot of ways to 'tostring' a 'nugetframework' and encapsulating "ours" seems quite nice.